MozziGuts_impl_MBED.hpp

```diff
@@ -112,7 +155,8 @@ int bufpos = 0;
 
 inline void commitBuffer(Sample buffer[], AdvancedDAC &dac) {
   SampleBuffer dmabuf = dac.dequeue();
-  for (int i = 0; i < dmabuf.size(); ++i) memcpy(dmabuf.data(), buffer, CHUNKSIZE*sizeof(Sample));
+  for (unsigned int i=0;i<CHUNKSIZE;i++) memcpy(dmabuf.data(), buffer, CHUNKSIZE*sizeof(Sample));
+  //for (unsigned int i=0;i<CHUNKSIZE;i++) dmabuf[i]=buffer[i];
```

Comment:
Still not sure what is the best *for now*…

- `memcpy(dmabuf.data(), buffer, CHUNKSIZE*sizeof(Sample));` is the logical choice, but it crashing here
- `for (unsigned int i=0;i<CHUNKSIZE;i++) memcpy(dmabuf.data(), buffer, CHUNKSIZE*sizeof(Sample));` seems to work here, but it is not satisfying to repeat the same operation over and over hoping for the cache to be flushed.
- `for (unsigned int i=0;i<CHUNKSIZE;i++) dmabuf[i]=buffer[i];` is mid-way, not repetitive but not really satisfying, but seems to work here…

Reply:
Do we keep that like this? I suggest adding a clear comment saying that this should change at some point (so that we do not forget).